parboiled-core/src/main/scala/org/parboiled2/support/OpTreeContext.scala

```diff
@@ -602,7 +602,7 @@ trait OpTreeContext[OpTreeCtx <: ParserMacros.ParserContext] {
         case Left(_)  ⇒ super.render(wrapped)
         case Right(x) ⇒ q"$x ne null"
       }
-    def renderInner(wrapped: Boolean) = call.asInstanceOf[Left[OpTree, Tree]].a.render(wrapped)
+    def renderInner(wrapped: Boolean) = call.left.get.render(wrapped)
```

Comment:
The reason we had the `asInstanceOf[Left[OpTree, Tree]]` was that we want to avoid the extra allocation for the projection instance here. So I'd say we should simply replace the `.a` with `.value` but keep the `asInstanceOf`.

Reply:
Scala 2.10 and 2.11 does not have the `value` method.
the `a` method removed since Scala 2.13